src/LinkDotNet.Blog.Web/Features/Components/ShortBlogPost.razor

```diff
@@ -33,11 +36,13 @@
 			</ul>
 		</div>
 		<div class="description">
-			<h1>@BlogPost.Title</h1>
-			<h2></h2>
+			<div class="header">
+				<h1>@BlogPost.Title</h1>
+				<BookmarkButton IsBookmarked="isBookmarked" OnClick="async () => { await BookmarkService.ToggleBookmark(BlogPost.Id); StateHasChanged(); }"></BookmarkButton>
```

Comment:
As this does more than one thing, it might be nice to have it inside a function rather than a lambda here.